scale-tests/scale_tests_utils.py

```diff
@@ -71,13 +66,19 @@ def get_service_options(service_name: str, service_account_info: typing.Dict,
     else:
         log.info("No options specified. Using defaults")
 
+    if service_name[0] != "/":
+        service_name = "/" + service_name
```

Comment:
I _think_ this is no longer required but doesn't hurt to make explicit

Reply:
Aye.